tests/solids/diamondC_2x1x1-Gaussian_pp_cplx_MSD_excited/CMakeLists.txt

```diff
@@ -0,0 +1,172 @@
+
+ IF (NOT QMC_CUDA)
+   IF (QMC_COMPLEX)
+
+# VMC short run for 1st excited state with no Jastrows 
+# Compare directly to sCI Energy of: E=-20.6068860259079
+#  LocalEnergy           =         -20.55017 +/-          0.00093
```

Comment:
These don't match at all.  Why?

Reply:
Aha!! I am blaming the locality error   in this case... but I am not sure why yet. We would need to run all electrons to some extend to understand a bit more...

Reply:
This is VMC though...

Reply:
VMC no Jastrow should always agree, regardless of Hamiltonian, etc.

Reply:
The issue was found.

Current workflow is as follow:
Pyscf generates 2 electron integrals (3idx+1 array for density fitting). MO coeffs generated with density fitting (loss of accuracy. 
then 3idx transformed to 4idx (recovering full accuracy in QP), where MSD energy is computed but MO_coeff are not required. This means in QMCPACK we run with under-converged MO_coeffs but MSD coefficients well converged. So energies don;t match.  Only way for all to match is either to generate the orbitals without DF (which is only possible for Molecules in pyscf or QP2), or, for solids by generating Natural orbitals in   QP, since then, MO_coeffs and MSD_dets are computed at the same accuracy.
Following tests are done solely with natural orbitals.

Reply:
Please at comments in the file to this effect.  The mismatched references are confusing.

Reply:
Changed the test by running from Natural orbitals. This way there is a consistent match